utils/kind-with-registry.sh

```diff
@@ -67,7 +67,7 @@ nodes:
 containerdConfigPatches:
 - |-
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
-    endpoint = ["http://${reg_host}:${reg_port}"]
+    endpoint = ["http://${reg_host}:5000"]
```

Comment:
Hi, @tao12345666333 @lingsamuel , I update this port, it should be fixed 5000 port.

Reply:
> Hi, @tao12345666333 @lingsamuel , I update this port, it should be fixed 5000 port.

why it should be 5000?

Reply:
@tao12345666333 the endpoint will be visit by k8s node which kind created, so the port should be the one that `registry` exposed (which is 5000 fixed), not the one that publish on our host. 
you can also refer to the [kind doc](https://kind.sigs.k8s.io/docs/user/local-registry/), and pay attention to this part
![image](https://user-images.githubusercontent.com/18495995/201085114-fe627fe6-e839-4a09-b77a-91b9d853a135.png)

Reply:
okay, I miss it.